Client/src/Components/Inputs/Field/index.jsx

```diff
@@ -185,10 +186,11 @@ const Field = forwardRef(
 						),
 					}}
 				/>
-				{error && (
+				{ error && (
 					<Typography
 						component="span"
 						className="input-error"
+						hidden={className? true: false}						
```

Comment:
Why do we have error message hidden or not based on having a className?

Reply:
Sorry, that is not clear for me, can you explain?

Reply:
If you use for threshold infrastructure scenario, then you need that class name and hide the error for each field

Reply:
Classname is not specified in jsdocs. The way it is, as I understand this, if we instantiate this compoennt passing a className to it, to extend styling maybe(?) we will hide the errors.